mobile-browser-based-version/server/run_server.js

```diff
@@ -19,10 +22,46 @@ const peerServer = ExpressPeerServer(server, {
 app.use('/deai', peerServer);
 
 
+let peers = [];
+function eventsHandler(request, response, next) {
+    const headers = {
+      'Content-Type': 'text/event-stream',
+      'Connection': 'keep-alive',
+      'Cache-Control': 'no-cache'
+    };
+    response.writeHead(200, headers);
+
+    let peerId = request.params['id']
+    console.log(peerId)
+    const data = `data: ${JSON.stringify(topology.getNeighbours(peerId))}\n\n`;
+
+    response.write(data);
+  
+    const newPeer = {
+      id: peerId,
+      response
+    };
+  
+    peers.push(newPeer);
+  
+    request.on('close', () => {
+      console.log(`${peerId} Connection closed`);
+      peers = peers.filter(peer => peer.id !== peerId);
+    });
+  }
+  
+  app.get('/neighbours/:id', eventsHandler);
```

Comment:
Something's off with the indentation here. Got a bit confused thinking that this was inside `eventsHandler`.